.gitignore

```diff
@@ -57,3 +57,6 @@ docs/_build/
 
 # OSX Files
 *.DS_Store
+
+# ANAF Data
+l10n_ro_vat_on_payment/data/
```

Comment:
@StefanRijnhart Can we add odoo parameter in .gitignore, i don't know if we can do it...i will update the path where the files from Fiscal Authorities will be downloaded in the tools.config['data_dir'], and remove the .gitignore...